src/studio/screens/console.c

```diff
@@ -926,7 +929,20 @@ static void onLoadCommandConfirmed(Console* console)
                 }
                 else printError(console, "\nfile not found");
 #else
-                printError(console, "\ncart loading error");
+                if(tic_project_ext(name)) {
+                    printError(console, "\nproject loading error");
+                    printFront(console, "\nThis version only supports binary cartridges (GIF, PNG, or TIC).");
```

Comment:
We don't have GIF cartridges :)
Only, PNG and TIC

Reply:
Aren't their own GIF cartridge out there?

Reply:
Never heard.